codecept.conf.js

```diff
@@ -0,0 +1,32 @@
+exports.config = {
+  tests: './tests/*_test.js',
+  output: './output',
+  helpers: {
+    Puppeteer: {
+      url: 'https://8s5ru.csb.app',
```

Comment:
We'll need this configured by a few things -- some tests will be on different urls. Some environments we're testing will use different urls. For now this is fine